frontend/src/screens/Authentication/constants.tsx

```diff
@@ -0,0 +1,7 @@
+export const API_BASE_URL = 'http://localhost:5000';
+export const ACCESS_TOKEN = 'accessToken';
+
+export const OAUTH2_REDIRECT_URI = 'http://localhost:3000/oauth2/redirect';
+
+export const GOOGLE_AUTH_URL = `${API_BASE_URL}/oauth2/authorize/google?redirect_uri=${OAUTH2_REDIRECT_URI}`;
+export const FACEBOOK_AUTH_URL = `${API_BASE_URL}/oauth2/authorize/facebook?redirect_uri=${OAUTH2_REDIRECT_URI}`;
```

Comment:
Remove prefixes

Reply:
Here I also can't remove prefixes due to href and proxy